recipes/dsm2dtm/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "dsm2dtm" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/seedlit/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 64a08fd43eea5c98cd7b7d4426f7646ad989e607ceb104047348b75d2aa2584b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - git
+    - python
+    - pip
+    - setuptools
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - gdal >=3.0.*
+    - rasterio >=1.2.*
+    - numpy >=1.20.*
```

With `>=` dependencies, you shouldn't also have wildcards `*`. This will work fine (matching the github repo):
```suggestion
    - gdal >=3.0.4
    - rasterio >=1.2.5
    - numpy >=1.20.3
```

Alternatively, if you don't care about the bugfix versions, you could also change the last number to `0` rather than `*`.

---

Thanks for pointing this out, this is my mistake.